tests/test_validators_cerberus.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import absolute_import
+from unittest import TestCase
+
+from spidermon.contrib.validation import CerberusValidator
+from spidermon.contrib.validation import messages
+
+class TestCerberusValidator(TestCase):
+    def test_schema(self):
```

Comment:
The test method shoud contain the intent of it: `test_invalid_schema_provided` would be a better name and you can them remove the comments.

I believe that raising a `ValueError` exception is a good way to handle this scenario.